### PR TITLE
Create dmk246.json

### PR DESCRIPTION
**Pull request by dmk246**
@githubteacher adding pin to the map 
closes #001

### CODE REVIEW

File: _pins/dmk246.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+githubhandle :dmk246
```
Comment (brianamarie) [inline _pins/dmk246.json]: Add three hyphens on line one, so that this text is on line two. `---`

Comment (brianamarie) [inline _pins/dmk246.json]: Also, make sure that it is `githubHandle: dmk246`, with the spaces exactly in those positions. 😄

File: _pins/dmk246.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+latitude: 37.751
+logitude: -97.822
```
Comment (brianamarie) [inline _pins/dmk246.json]: Make sure that `longitude` is spelled correctly.

Comment (brianamarie) [inline _pins/dmk246.json]: Also, add three hyphens at the very bottom.

**Comment by brianamarie** [comment]
@dmk246 I've left a few line comments with suggestions on what to change so the tests will pass.  [Click "Help Me Troubleshoot" for some other specific ideas](https://services.github.com/on-demand/intro-to-github/edit-files-on-github), or [watch this video to get some help](https://vimeo.com/201011288). If you need or want help, let me know! If this hasn't been resolved by the end of the week, I'll go ahead and make the changes for you so that it can be merged in. 😄

File: _pins/dmk246.json (hunk 2 above)
Comment (dmk246) [inline _pins/dmk246.json]: HI @brianamarie Thank you for your help and for pointing out what needs fixed.  I have a few questions.  First .. I followed the directions in the tutorial on how to edit and make changes... and when I click on the pencil icon like it tells me to under files changed... it tells me that I need to be signed in (which i am ) or I need to have push access to dmk246-pgh branch, which is the branch that I created so wouldn't I have the privileges?  Also, I am not sure I know what you mean about the hypens.  So literally just add them above the first line of text?  Sorry... I am completely new to all of this, even coding. So I know they are probably really silly mistakes.  I am really brand new to this world!  Thanks again for your help and patience.

Comment (brianamarie) [inline _pins/dmk246.json]: Hi @dmk246! A few things...

1. I don't know why or how, but somehow you got taken off as a collaborator for this repository. Once you [go and confirm your invitation](https://github.com/githubschool/open-enrollment-classes-introduction-to-github/invitations), you should be able to edit again. So sorry about this! 

2. Yes, you'll have the hyphens so the file will literally look like this: 
screen shot 2017-08-09 at 2 32 02 pm

These aren't silly mistakes at all - one wasn't even at all your fault!!! The hardest part of learning Git and GitHub are these hiccups. The fact that you're here again trying it out means you're going to be spectacular at it. ✨ Let me know if you need or want any more help, I'm around!

**Comment by dmk246** [comment]
Thank you so much @brianamarie for your help and great explanations!  I made the changes... lets see how I do!!  Also, thanks for the encouragement ... it can be quite intimidating coming into all this... so everything helps so much!!

**Comment by brianamarie** [comment]
@dmk246 You are SO close. From a quick glance, it looks like there's one more small thing. Make sure the capitalization is `githubHandle`. The tests are very picky 😉

**Comment by dmk246** [comment]
@brianamarie OH geezz.. I am so sorry!!!  SO much for me to think about!!  Thank you AGAIN! : )

**Comment by brianamarie** [comment]
Yay! Looks great! Great job!

**Comment by dmk246** [comment]
@brianamarie ... OH MY .. WOOHOOO!@!!!  HAHAH .. I am celebrating over here!  Thank you so much ... it means so much that you took the time to help me out!

**Pull request merged**